models/examples/06_token_examples.py

```diff
@@ -0,0 +1,79 @@
+
+from credmark.cmf.model import Model
+from credmark.cmf.model.errors import ModelRunError
+from credmark.cmf.types import Address, NativeToken, Token
+from credmark.dto import DTO, DTOField
+from models.dtos.example import ExampleModelOutput
+
+
+class _TokenLoadingInput(DTO):
```

Comment:
why the _ before the name?

Reply:
leading _ hints that the class is internal.

Reply:
Interesting, ok